src/main/java/main/FigureSupplier.java

```diff
@@ -0,0 +1,69 @@
+package main;
+
+import core.basesyntax.Circle;
+import core.basesyntax.Figure;
+import core.basesyntax.IsoscelesTrapezoid;
+import core.basesyntax.Rectangle;
+import core.basesyntax.RightTriangle;
+import core.basesyntax.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random random = new Random();
+    private static final int DEFAULT_CIRCLE_RADIUS = 10;
+    private static final int MAX_INT = 25;
+    private static final String CIRCLE_COLOR = Colors.WHITE.toString().toLowerCase();
```

Comment:
```suggestion
    private static final String CIRCLE_COLOR = Colors.WHITE.name().toLowerCase();
```
when you want to get a string name of enum better use name()
because toString can be overridden and will return the same value for each enum